docs/source/samples/20_color_rotate_warp.rst

```diff
@@ -0,0 +1,22 @@
+20 - Color Rotate Warp
+======================
+
+This example shows usage of ImageWarp to crop a rotated rectangle area on a frame, or perform various image transforms: rotate, mirror, flip, 
```

Comment:
I assume this should be: `This example shows usage of **ImageManip** to crop a rotated...`. Please also fix in python script (demo 20)